### ISSUE TITLE
Calendar object doesn't work for screen readers

### ISSUE THREAD

**Issue by ellvix** [LRNWebComponents/hax-body#155]
From a screen reader perspective, the whole calendar object is broken. It only reads the numbers, and is difficult to figure out what the context is and what's going on at all. It's bad enough that there is no simple fix; a whole rewrite or new calendar would be best.

**Comment by btopro** [comment]
tagging w/ https://github.com/LRNWebComponents/lrn-calendar as that's where that project comes from. This will need a bunch of work as it was an experiment from a student employee. This is a known thing (needing a full rewrite) but thank you for confirming this :)
